src/components/dialog.ts

```diff
@@ -422,6 +435,15 @@ export class Dialog {
     this.hidden_ = false;
   }
 
+  /** Supresses click events, unless the window is closable. */
```

Comment:
This could be clarified, as the implementation always stops click events, but if the dialog is closeable it closes the dialog on click.